docs/cudf/source/conf.py

```diff
@@ -23,11 +22,6 @@
 from docutils.nodes import Text
 from sphinx.addnodes import pending_xref
 
-import cudf
```

Comment:
How are the docs built without importing the cudf module? I thought that was required for autodoc and other Sphinx features to work but maybe I’m wrong.

Reply:
[These logs](https://github.com/rapidsai/cudf/actions/runs/4237425138/jobs/7364044147) suggest that the docs build successfully without the `cudf` import.

Reply:
See my commit message in the commit below for additional details

- https://github.com/rapidsai/cudf/pull/12592/commits/70baa0ffb9a495730068d10d4c3dbecbe8565d76

Reply:
Sweet. Thanks for verifying!